src/Sulu/Bundle/AdminBundle/Resources/js/components/AutoComplete/AutoComplete.js

```diff
@@ -0,0 +1,182 @@
+// @flow
+import React from 'react';
+import type {ChildrenArray, Element, ElementRef} from 'react';
+import {observer} from 'mobx-react';
+import {action, computed, observable} from 'mobx';
+import debounce from 'debounce';
+import Input from '../Input';
+import Popover from '../Popover';
+import Menu from '../Menu';
+import Suggestion from './Suggestion';
+import autoCompleteStyles from './autoComplete.scss';
+
+const LENS_ICON = 'search';
+const DEBOUNCE_TIME = 250;
+const POPOVER_HORIZONTAL_OFFSET = 5;
+const POPOVER_VERTICAL_OFFSET = -2;
+
+type Props = {
+    children: ChildrenArray<Element<typeof Suggestion>>,
+    /** The value inside the input field */
+    value: string,
```

Comment:
I'd argue that this value has to be one of the possible options, and not what's currently in the input field. What's in the input field could be another observable.